src/stirling/bpf_tools/bcc_wrapper.h

```diff
@@ -89,7 +89,7 @@ class BCCWrapper {
     return task_struct_offsets_opt_;
   }
 
-  virtual ebpf::BPF& BPF() = 0;
+  virtual ebpf::BPF* BPF() = 0;
```

Comment:
What's the rationale for why this change is needed?

Reply:
It will be easier to override this way. The replaying BCC wrapper will not have a BPF instance to return. I tried a status-or containing a reference and that is unsupported (if I've got that right...).